handsontable/src/3rdparty/walkontable/test/unit/cell/range.unit.js

```diff
@@ -822,77 +865,256 @@ describe('CellRange', () => {
+
+        expect(range.isSingle()).toBe(false);
+      }
+    });
+  });
+
+  describe('isSingleCell()', () => {
+    it('should return `true` when `from` and `to` are equals and there are no headers selected', () => {
       {
         const range = createRange(0, 0, 4, 5, 4, 5);
 
-        expect(range.isSingle()).toBe(true);
+        expect(range.isSingleCell()).toBe(true);
       }
       {
         const range = createRange(0, 0, 0, 0, 0, 0);
 
-        expect(range.isSingle()).toBe(true);
+        expect(range.isSingleCell()).toBe(true);
       }
     });
 
-    it('should return `false` when `from` and `to` are equals and there is header selected', () => {
+    it('should return `false` when `from` and `to` are equals and there are some headers selected', () => {
       {
         const range = createRange(0, 0, -1, 0, -1, 0);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
       {
         const range = createRange(0, 0, 0, -1, 0, -1);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
       {
         const range = createRange(0, 0, 0, -1, -1, 0);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
     });
 
     it('should return `false` when `from` and `to` are not equal', () => {
       {
         const range = createRange(0, 0, 0, 0, -1, 0);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
       {
         const range = createRange(0, 0, 0, 0, 0, 1);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
       {
         const range = createRange(0, 0, 0, 0, 1, 0);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
       {
         const range = createRange(0, 0, 0, 0, 1, 1);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
       {
         const range = createRange(0, 0, -1, 0, 0, 0);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
       {
         const range = createRange(0, 0, 0, 1, 0, 0);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
       {
         const range = createRange(0, 0, 1, 0, 0, 0);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
       }
       {
         const range = createRange(0, 0, 1, 1, 0, 0);
 
-        expect(range.isSingle()).toBe(false);
+        expect(range.isSingleCell()).toBe(false);
+      }
+    });
+  });
+
+  describe('isSingleHeader()', () => {
+    it('should return `false` when `from` and `to` are equals and there are no headers selected', () => {
+      {
+        const range = createRange(0, 0, 4, 5, 4, 5);
```

Comment:
`createRange` could be used in one line with `expect`, but I don't mind. It looks clear 👍  Nice :)